<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>VMI Ramper</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>Select Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="config_select_cb"/>
       </item>
       <item>
        <widget class="QPushButton" name="config_load_btn">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load Config</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Global Ramp Rate (%/s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ramp_rate_le"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Scale Voltages Factor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scale_voltage_le"/>
       </item>
       <item>
        <widget class="QPushButton" name="set_scale_btn">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="device_table">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Save Config As:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="config_save_le"/>
       </item>
       <item>
        <widget class="QPushButton" name="save_config_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save Config</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="ramp_bn">
         <property name="text">
          <string>Start Ramp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ramp_0_bn">
         <property name="text">
          <string>Ramp all To 0V</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
